src/content/docs/agile-handbook/heroing/what-is-a-hero.mdx

```diff
@@ -81,39 +81,80 @@ Common questions and requests include:
-      - If the writer closes the ticket, they must leave a comment explaining why.
-    - Writer needs to add: 
-      - Acceptance criteria
-      - User story
-      - Action items
-      - Resources 
-    - While rewriting, focus on making the ticket as swarmable as possible
-    - Assign a priority [See priority info in our team's `DOC Jira and Github fields - Project 401 - 2022` doc]
-    - Add the following label: `doc_groomed`
-3. Triaged tickets enter sprints in one of two ways:
-    - **Scrum leaders**: Bring in tickets based on marked priority
-    - **Liaisons**: Bring in high value issues
-    - **Squads**: During backlog grooming the team can chat about bringing in tickets to fill out a sprint
-
 <Callout variant="important">
 For writers in Europe:
 
 - Route incoming issues to the liaison as you have time. As writers in Europe are "eternal heroes" it isn’t fair to have you do this every day.
 - You should still triage issues routed to you from your liaisonships once per day.
  </Callout>
 
+## Customer feedback sidekick [#cf_sidekick]
+
+Customer feedback from our docs site creates tickets in Jira. It's the sidekick's responsibility to look at every ticket that comes in (usually about 5-10 per day) and make decisions about each one.
+
+We have three options for these tickets:
+
+1. Close it.
+2. Fix it.
+3. Backlog it.
+
+<Callout variant="important">
+Unlike the hero, we don't take any points out of our sprint for the sidekick work. If you're spending more than 60 minutes per day on customer feedback tickets, that's too much time! Please let your manager know.
+ </Callout>
+
+### Close customer feedback
+
+Most customer feedback that comes in isn't actionable, either because it verges on nonsense or because the feedback is too non-specific to act on.
+
+For example:
+
+`These docs are bad.`
+
+or 
+
+`I can't log in to Netflix.`
+
+Follow this process to close these:
+
+1. Add a category label based on the URL, such as `doc_apm` or `doc_errors-inbox`.
+2. Set the Workflow status to **Closed** 
+3. Set the Resolution to **Cannot Reproduce**, **Filed by Mistake**, or **Won't Do** depending on what feels most appropriate.
+4. Add a brief note about why you closed it. Usually, "Not actionable" is plenty.
+
+### Fix customer feedback
+
+Sometimes, though rarely, a customer feedback ticket comes in that's highly actionable and is a quick fix, such as a broken link or an outdated image. 
+
+Follow this process to close these:
+
+1. Create a PR to address the issue.
+2. Add a category label based on the URL, such as `doc_apm` or `doc_errors-inbox`.
+3. Set the Workflow status to **Closed** 
+4. Set the Resolution to **Done**.
+5. Add a link to the PR.
+6. Work the PR to its merge at your leisure.
+
+### Backlog customer feedback
+
+Our goal is to never have more than 13 customer feedback tickets in our backlog. If you're adding number 14, please check the other tickets to see if there's an old one that you can close or add to a sprint. 
```

Comment:
I don't think this 13 should be this firm FWIW. Perhaps get into the logic more?

> We try not to have more than 4–6 weeks of customer feedback tickets open. In practice, that means around 13 tickets in our backlog total...

or similar

Reply:
@austin-schaefer I think we need to set an upper bound on how many customer feedback tickets we have in our backlog. It's completely arbitrary, of course. I think setting a hard number provides helpful clarity. But I'm extremely agnostic on what number we pick. I do think it's good to live in reality though. Is it realistic to think we'll ever meaningfully address 50 customer feedback tickets? I don't think so. What's a number that we could meaningfully address in the short term? 5-15 I'd say, depending on how much we want to prioritize this work over other work.

Reply:
It's hard to gauge though, right? My sense of things right now is that our team capacity is 5-15 customer feedback tickets, depending on the scope of each of those tickets

Reply:
I've tried to describe this in a more general way that I think gets at what I tried to achieve with the specific numbers.